analysis/clinvar/clinvar_variation_analysis.ipynb

```diff
@@ -3312,25 +3316,25 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>Normalized</th>\n",
-       "      <td>2198936</td>\n",
+       "      <td>3745114</td>\n",
        "      <td>0</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>Not Normalized</th>\n",
-       "      <td>1041</td>\n",
-       "      <td>10650</td>\n",
+       "      <td>6749</td>\n",
+       "      <td>5604</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
        "                Supported  Not Supported\n",
-       "Normalized        2198936              0\n",
-       "Not Normalized       1041          10650"
+       "Normalized        3745114              0\n",
+       "Not Normalized       6749           5604"
```

Comment:
@jarbesfeld IIRC you had a question on if these were right... can you remind me what that was again?